core/src/create.jl

```diff
@@ -127,6 +149,42 @@ function Basin(db::DB, config::Config)::Basin
     )
 end
 
+function Control(db::DB, config::Config)::Control
+    condition = load_structvector(db, config, ControlConditionV1)
+
+    condition_value = fill(false, length(condition.node_id))
+    control_state::Dict{Int, Tuple{String, Float64}} = Dict()
+
+    rows = execute(db, "select from_node_id, edge_type from Edge")
+    for (; from_node_id, edge_type) in rows
+        if edge_type == "control"
+            control_state[from_node_id] = ("undefined_state", 0.0)
+        end
+    end
+
+    logic = load_structvector(db, config, ControlLogicV1)
+
+    logic_mapping::Dict{Tuple{Int, String}, String} = Dict()
```

Comment:
I think you refactored this back, why not
`logic_mapping = Dict{Tuple{Int, String}, String}()`?